### PR TITLE
Fix typos.

### PR DESCRIPTION
**Pull request by haosdent**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:

/kind cleanup

**What this PR does / why we need it**:

Make comments clear.

**Which issue(s) this PR fixes**:

Fixes typos

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/proxy/config/config.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ type EndpointsHandler interface {
 	// OnEndpointsUpdate is called whenever modification of an existing
 	// endpoints object is observed.
 	OnEndpointsUpdate(oldEndpoints, endpoints *v1.Endpoints)
-	// OnEndpointsDelete is called whever deletion of an existing endpoints
+	// OnEndpointsDelete is called whenever deletion of an existing endpoints
```
Comment (atoato88) [inline pkg/proxy/config/config.go]: There is a same typo at line 559 of `pkg/proxy/iptables/proxier.go`.

Comment (haosdent) [reply pkg/proxy/config/config.go]: got it, let me fix

**Review by fejta — COMMENTED** [review]
We have two jobs that attempt to catch spelling mistakes. Please update the automation so that we catch these misspellings in the future.

https://github.com/kubernetes/kubernetes/blob/master/hack/verify-shellcheck.sh
https://github.com/kubernetes/kubernetes/blob/master/hack/verify-spelling.sh

File: pkg/proxy/config/config.go (hunk 1 above)
Comment (haosdent) [reply pkg/proxy/config/config.go]: @atoato88 just fixed, sry for delay. Could you help to review again?

**Comment by haosdent** [comment]
@fejta 
```
$ echo 'whever' >/tmp/test_2.log
$ misspell /tmp/test_2.log
$ echo $?
0
```
Looks like misspell consider `whever` is valid word as well.

**Comment by haosdent** [comment]
Same to `succeessful`. 
```
$ echo "succeessful" | misspell -w -q
succeessful
```

**Comment by haosdent** [comment]
@fejta do you mean need to modify https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/client9/misspell/words.go ? Sorry a bit lost at here.

**Review by logicalhan — COMMENTED** [review]
/lgtm

The spelling changes look okay to me.

File: pkg/proxy/config/config.go (hunk 1 above)
Comment (atoato88) [reply pkg/proxy/config/config.go]: @haosdent Thank you for fixing typo in `pkg/proxy/iptables/proxier.go`.
It's ok for me at point I commented.
Please follow comments from other member.

Comment (haosdent) [reply pkg/proxy/config/config.go]: Got it, thx @atoato88 's help Orz

**Review by neolit123 — COMMENTED** [review]
/approve
/priority backlog

**Comment by fejta** [comment]
> do you mean need to modify https://github.com/kubernetes/kubernetes/blob/master/vendor/github.com/client9/misspell/words.go

Yes, I would like to see PRs ensure:
a) that all instances in the repo are fixed
b) a presubmit test prevents someone from reintroducing this error in the future

**Review by timothysc — APPROVED** [review]
/lgtm 
/approve

**Pull request merged**